docs/deployment.md

```diff
@@ -8,10 +8,10 @@ That said, the platform does require some server-side technologies to *build* th
 
 ## Automation services
 
-The easiest way to perform these builds is to use a free automation service like TravisCI or CircleCI. Alternatives such as self-hosted Jenkins are possible, but require more maintenance and likely would not be free.
+The easiest way to perform these builds is to use a free automation service like Github Actions, TravisCI, or CircleCI. Alternatives such as self-hosted Jenkins are possible, but require more maintenance and likely would not be free.
 
 > Note that the "starter" repositories are pre-configured for to handle all of
```

Comment:
Think this should say
`> Note that the "starter" repositories are pre-configured to handle all of`
instead of
> Note that the "starter" repositories are pre-configured **for** to handle all of